src/IECoreScene/Camera.cpp

```diff
@@ -44,6 +44,42 @@ using namespace IECoreScene;
 using namespace Imath;
 using namespace std;
 
+namespace {
+
+	// TODO - it's a bit ridiculous that this helper function isn't just declared in CompoundDataBase,
```

Comment:
I don't recall the exact conversation, but I think a more accurate reflection of my opinion at the time was that there would be a lot of considerations which are outside the scope of this already massive PR. So yeah, great points, but lets circle back to this later rather than holding up the cameras any longer. We could probably change the todo to something more like `// \todo: Remove once we have similar utilities on CompoundData`